### PR TITLE
Format after protected modifer

### PR DESCRIPTION
**Pull request by DickvdBrink**
Formatting after the modifier didn't work correctly.
Hope you like it :)

### CODE REVIEW

File: src/services/formatting/rules.ts [hunk 1]
```diff
@@ -318,7 +318,7 @@ namespace ts.formatting {
...
+            this.SpaceAfterCertainTypeScriptKeywords = new Rule(RuleDescriptor.create4(Shared.TokenRange.FromTokens([SyntaxKind.ClassKeyword, SyntaxKind.DeclareKeyword, SyntaxKind.EnumKeyword, SyntaxKind.ExportKeyword, SyntaxKind.ExtendsKeyword, SyntaxKind.GetKeyword, SyntaxKind.ImplementsKeyword, SyntaxKind.ImportKeyword, SyntaxKind.InterfaceKeyword, SyntaxKind.ModuleKeyword, SyntaxKind.NamespaceKeyword, SyntaxKind.PrivateKeyword, SyntaxKind.PublicKeyword, , SyntaxKind.ProtectedKeyword, SyntaxKind.SetKeyword, SyntaxKind.StaticKeyword]), Shared.TokenRange.Any), RuleOperation.create2(new RuleOperationContext(Rules.IsSameLineTokenContext), RuleAction.Space));
```
Comment (DanielRosenwasser) [inline src/services/formatting/rules.ts]:
````
SyntaxKind.PrivateKeyword, SyntaxKind.PublicKeyword, , SyntaxKind.ProtectedKeyword`
                                                    ^^
```

There is an implicit undefined in this list.

Comment (DickvdBrink) [inline src/services/formatting/rules.ts]: Oops sorry, will force-push

Comment (DanielRosenwasser) [inline src/services/formatting/rules.ts]: Sure thing!

As a side note, for bigger changes, we prefer to keep commits around for the purpose of review history. Clearly it's not an issue here, but thought I'd just bring that up.

**Comment by DanielRosenwasser** [comment]
Thanks @DickvdBrink!

**Comment by mhegazy** [comment]
@aozgaa you will need to add `abstract` to the list just like @DickvdBrink did, and @rbuckton will need to add `async` and `await` as well.

**Comment by mhegazy** [comment]
@DickvdBrink we should add `default` as well. e.g. `export default class Foo {}`

**Comment by vladima** [comment]
@mhegazy formatting for `abstract` is already included in corresponding PR

**Comment by DickvdBrink** [comment]
Will do later today @mhegazy (with some tests as well :))

**Pull request merged**